### PR TITLE
fix: ndarray to array

### PR DESCRIPTION
**Pull request by qfeuilla**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "statusBarItem.hoverBackground": "#c1ab07",
+    "statusBarItem.remoteBackground": "#f2d709",
+    "statusBarItem.remoteForeground": "#15202b",
+    "titleBar.activeBackground": "#f2d709",
+    "titleBar.activeForeground": "#15202b",
+    "titleBar.inactiveBackground": "#f2d70999",
+    "titleBar.inactiveForeground": "#15202b99"
+  },
+  "peacock.color": "#f2d709"
+}
```
Comment (madvid) [inline .vscode/settings.json:22]: has to be removed

**Review by madvid — CHANGES_REQUESTED** [review]
It is incorrect to change numpy.ndarray by numpy.array. numpy.array is a function whereas numpy.ndarray is a class, thus we can have instance of this class

[numpy.ndarray](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html)
[numpy.array](https://numpy.org/doc/stable/reference/generated/numpy.array.html?highlight=numpy%20array#numpy.array)

File: .vscode/settings.json (hunk 1 above)
Comment (qfeuilla) [reply .vscode/settings.json:22]: same comment as in bootcamp_ml

**Comment by qfeuilla** [comment]
indead it is a mistake to change ndarray to array, my bad

**Pull request closed without merging**